README.md

```diff
@@ -86,11 +86,12 @@ The following tables lists the main configurable parameters of the `deephealth-f
 | `backend.clientId`  | OAuth Client ID that you can get registering the front-end at <backend-host>/backend/auth/applications/  | `nil`              |
 | `service.type`      | Kubernetes service type of the front-end                                                                 | `NodePort`         |
 | `ingress.enabled`   | Enable the ingress for the front-end service                                                             | `false`            |
-| `ingress.annations` | Annotations for the ingress related with the front-end service                                           | {}                 |
+| `ingress.annations` | Annotations for the ingress related with the front-end service                                           | `{}`               |
```

Comment:
oops...there's a typo here.  That should be `ingress.annotations`, not `annatations`. From a quick look it seems that the error is only here in the readme. The template [uses the correct spelling`](https://github.com/deephealthproject/docker-frontend/blob/master/k8s/deephealth-frontend/templates/ingress.yaml#L12)

Reply:
Ok, I'll fix it!

Reply:
Fixed in https://github.com/deephealthproject/docker-frontend/commit/262505aced7e1e9ae49c60abcda3e28141eddec9